### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8627,6 +8628,7 @@ hwile->while
 hwole->whole
 hybernate->hibernate
 hydogen->hydrogen
+hydrolics->hydraulics
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the singular

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8644,6 +8644,7 @@ hundreths->hundredths
 hundrets->hundreds
 hunrgy->hungry
 husban->husband
+hussel->hustle
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add mussel?

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**